README.md

```diff
@@ -32,6 +32,13 @@ brew install desktop-file-utils shared-mime-info       \
 
 
 ```sh
-meson setup build
-meson compile -C build
+git clone "https://github.com/gtkwave/gtkwave.git"
+cd gtkwave
+meson setup build && cd build && meson install
 ```
+
+### Running GTKWave
+```sh
+gtkwave [path to a .vcd, .fst, .ghw dump file or a .gtkw savefile]
+```
+For more information about available command line parameters refer to the built in-help (`gtkwave --help`) or the (GTKWave docs)[https://gtkwave.github.io/gtkwave/man/gtkwave.1.html].
```

Comment:
You did mix up the bracket types in the Markdown, which did break the link:
```suggestion
For more information about available command line parameters refer to the built in-help (`gtkwave --help`) or the [`gtkwave` man page](https://gtkwave.github.io/gtkwave/man/gtkwave.1.html).
```
In my suggestion I've changed "docs" to "man page", which not only serves as a link to the relevant docs but also as a reminder that man pages still exist. I'm not sure if many users still prefer to read the man page over the online version, but it doesn't hurt to mention it.